polaris-react/src/components/Box/Box.stories.tsx

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import type {ComponentMeta} from '@storybook/react';
+import {Box, Icon} from '@shopify/polaris';
+import {PaintBrushMajor} from '@shopify/polaris-icons';
+
+export default {
+  component: Box,
+} as ComponentMeta<typeof Box>;
+
+export function Default() {
```

Comment:
Do we need a default example if this isn't really a default? Default wouldn't render anything visual except children

Reply:
Storybook complains if there's no `Default`. But if you know a workaround, I agree this `Box` default is awkward. Maybe I should remove the background from Default and let it render nothing?

Reply:
Ah right, yeah maybe let's just render the children and show that there's nothing visual that renders when no props are provided 👍